### PR TITLE
Null safety

### LINKED ISSUE #122: Migrate the package to null safety please.
Any news? :)

### PR DESCRIPTION
**Pull request by daadu**
Closes #122

### CODE REVIEW

**Comment by daadu** [comment]
Added pre-release [v0.3.0-nullsafety.0 on pub.dev](https://pub.dev/packages/wifi_iot/versions/0.3.0-nullsafety.0)

File: example/pubspec.yaml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 name: wifi_iot_example
 description: Demonstrates how to use the wifi_iot plugin.
 
+environment:
+  sdk: '>=2.10.0 <3.0.0'
```
Comment (mavyfaby) [inline example/pubspec.yaml:5]: To clarify is it `2.10.0` or `2.12.0`? since the null-safety are available in version `>= 2.12.0` of dart.

**Review by mavyfaby — CHANGES_REQUESTED** [review]

File: example/pubspec.yaml (hunk 1 above)
Comment (daadu) [reply example/pubspec.yaml:5]: example code is not null-safe yet.

Comment (daadu) [reply example/pubspec.yaml:5]: example app barely works, needs some work.

**Comment by diegotori** [comment]
LGTM for the library implementation.

File: example/pubspec.yaml (hunk 1 above)
Comment (mavyfaby) [reply example/pubspec.yaml:5]: I'll contribute.

Comment (mavyfaby) [reply example/pubspec.yaml:5]: LGTM.. I'll make a new PR once this PR merged.

**Review by mavyfaby — APPROVED** [review]

**Comment by daadu** [comment]
Release [v0.3.0 on pub.dev](https://pub.dev/packages/wifi_iot/versions/0.3.0)

**Pull request merged**